app/controllers/app.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'roda'
 require 'slim'
-
+require_relative '../infrastructure/gateways/api'
```

Comment:
It's weird that we need to add this line so that our application can run. Otherwise, it will occur uninitialized constant error for THSR: Api